modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -802,48 +802,60 @@ protected function content_template() {
 		?>
 		<div class="e-n-accordion" role="tablist" aria-orientation="vertical">
 			<# if ( settings['items'] ) {
-			const elementUid = view.getIDInt().toString().substr( 0, 3 ),
+			const elementUid = view.getIDInt().toString().substring( 0, 3 ),
 				titleHTMLTag = elementor.helpers.validateHTMLTag( settings.title_tag ),
+				itemTitleText = 'item-title-text-' + elementUid,
 				defaultState = settings.default_state,
-				itemTitleIcon = elementor.helpers.renderIcon( view, settings['accordion_item_title_icon'], { 'aria-hidden': true }, 'i' , 'object' ) ?? '',
+				itemTitleIcon = elementor.helpers.renderIcon( view, settings['accordion_item_title_icon'], { 'aria-hidden': true }, 'i', 'object' ) ?? '',
 				itemTitleIconActive = '' === settings.accordion_item_title_icon_active.value
 					? itemTitleIcon
-					: elementor.helpers.renderIcon( view, settings['accordion_item_title_icon_active'], { 'aria-hidden': true }, 'i' , 'object' );
+					: elementor.helpers.renderIcon( view, settings['accordion_item_title_icon_active'], { 'aria-hidden': true }, 'i', 'object' );
 			#>
 
-			<# _.each( settings['items'], function( item, index ) {
-			const itemCount = index + 1,
-				itemUid = elementUid + itemCount,
-				itemWrapperKey = itemUid,
-				itemTitleKey = 'item-' + itemUid;
-
-			if ( '' !== item.element_css_id ) {
-				itemId = item.element_css_id;
-			} else {
-				itemId = 'e-n-accordion-item-' + itemUid;
-			}
-
-			const itemWrapperAttributes = {
-				'id': itemId,
-				'class': [ 'e-n-accordion-item','e-normal' ],
-			};
-
-			if ( defaultState === 'expanded' && index === 0 ) {
-				itemWrapperAttributes['open'] = true;
-			}
-
-			view.addRenderAttribute( itemWrapperKey,  itemWrapperAttributes);
-
-			view.addRenderAttribute( itemTitleKey, {
-				'class': [ 'e-n-accordion-item-title' ],
-			} );
-
-			#>
+				<# _.each( settings['items'], function( item, index ) {
+				const itemCount = index + 1,
+					itemUid = elementUid + itemCount,
+					itemWrapperKey = itemUid,
+					itemTitleKey = 'item-' + itemUid;
+
+					view.addRenderAttribute( itemTitleKey, {
+						'class': ['e-n-accordion-item-title'],
+					});
+
+					view.addRenderAttribute( itemTitleText, {
```

Comment:
I would move this to after `view.addRenderAttribute( itemTitleKey` to keep the original structure of this code.

Reply:
@hein-obox Good idea